src/main/webapp/help.html

```diff
@@ -6,17 +6,18 @@
     <link rel="stylesheet" href="assets/css/custom_style.css">
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
     <title>Help for Sections Manager</title>
+    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
+    <script type="text/javascript">
+	$('document').ready(function() {
+		 $("#buildInfo").load("build.txt");
+			});
+</script>
   </head>
   <body class="container-fluid" style="padding-top:20px;">
   	<h1>Help for Sections Utility</h1>
-	  <p><span class="label label-danger">All of this content needs to be replaced with usage documentation when functionality is finished</span></p>
-	  <p>Version, revision, etc</p>
-	  <p>Contact info, bug reporting</p>
-	  <p>
-	      <strong>Available test users:</strong> amanjo, csev, jxiaapl, pbogart, pfbodary, raquelng, sileo, trohlck
-	  </p>
-	  <p>
-	      <strong>Available test users for xlisting:</strong> instx, instz (has courses in default term)
-	  </p>
+	  <p><a href="https://umichprod.service-now.com/kb_view.do?sysparm_article=KB0020309">ServiceLink knowledge base article</a></p>
```

Comment:
Is that link URL correct?  I get an error message: "> INSUFFICIENT ROLES TO VIEW PROTECTED ARTICLE".  The link should go to something accessible by all or the article's role requirements should be relaxed.